lms/djangoapps/course_home_api/toggles.py

```diff
@@ -73,3 +90,39 @@ def send_course_progress_analytics_for_student_is_enabled(course_key):
     Returns True if the course completion analytics feature is enabled for a given course.
     """
     return COURSE_HOME_SEND_COURSE_PROGRESS_ANALYTICS_FOR_STUDENT.is_enabled(course_key)
+
+
+def audit_learner_verified_preview_is_enabled(course_key):
+    """
+    Returns True if the audit learner verified preview feature is enabled for a given course.
+    """
+    return COURSE_HOME_AUDIT_LEARNER_VERIFIED_PREVIEW.is_enabled(course_key)
+
+
+def learner_can_preview_verified_content(course_key, user):
```

Comment:
Probably a good idea to cache this result somewhere? Not needed for this PR, but a suggested future development

Reply:
Have cached.